packages/ag-charts-website/src/content/docs/themes/index.mdoc

```diff
@@ -114,18 +144,18 @@ var myTheme: AgChartTheme = {
             axes: {
                 category: {
                     line: {
-                        stroke: 'gray',
+                        stroke: '#881008',
```

Comment:
Shouldn't these be removed now?

Reply:
Good spot, thanks! Replaced with `axisColor` and then added `width` instead so we still have an axis override.